examples/data/plant_meta.yml

```diff
@@ -1,43 +1,45 @@
 asset:
-  elevation: elevation_m
-  hub_height: Hub_height_m
+  elevation: elevation_m  # elevation abouve sea level, meters
+  hub_height: Hub_height_m  # hub height, meters
   id: Wind_turbine_name
-  latitude: Latitude
-  longitude: Longitude
-  rated_power: Rated_power
-  rotor_diameter: Rotor_diameter_m
+  latitude: Latitude  # WGS-84 latitude
+  longitude: Longitude  # WGS-84 longitude
+  rated_power: Rated_power  # rated power, MW
+  rotor_diameter: Rotor_diameter_m  # rotor diameter, meters
 curtail:
-  availability: availability_kwh
-  curtailment: curtailment_kwh
-  frequency: 10T
-  time: time
-latitude: 48.4497
-longitude: 5.5896
-capacity: 8.2
+  IAVL_DnWh: availability_kwh  # availability, kWh
+  IAVL_ExtPwrDnWh: curtailment_kwh  # curtailment, kWh
+  frequency: 10T  # timestamp frequency
+  time: time_utc  # timestamp
+latitude: 48.4497  # WGS-84 latitudinal plant centroid
+longitude: 5.5896  # WGS-84 longitudinal plant centroid
+capacity: 8.2  # Plant capacity, MW
 meter:
-  energy: net_energy_kwh
-  time: time
+  MMTR_SupWh: net_energy_kwh  # net energy, kWh
+  time: time_utc  # timestamp
 reanalysis:
-  era5:
-    frequency: H
-    surface_pressure: surf_pres
-    temperature: t_2m
-    time: datetime
-    windspeed_u: u_100
-    windspeed_v: v_100
-  merra2:
-    frequency: H
-    surface_pressure: surface_pressure
-    temperature: temp_2m
-    time: datetime
-    windspeed_u: u_50
-    windspeed_v: v_50
+  era5:  # reanalysis product name/ID
+    frequency: H  # timestamp frequency
+    WMETR_EnvPres: surf_pres  # surface pressure, Pa
+    WMETR_EnvTmp: t_2m  # temperature, K
+    time: datetime  # timestamps
+    WMETR_HorWdSpdU: u_100  # u-direction windspeed, m/s
+    WMETR_HorWdSpdV: v_100  # v-direction windspeed, m/s
+    WMETR_HorWdDir: winddirection_deg  # wind direction, degrees
```

Comment:
It's saved because it's already been computed, so for more efficient reloading to `PlantData` and for consistency of results across uses, having the data be the same each time would be ideal. The idea is to save the data that exists, not just to limit it to the columns originally provided or the required columns.